tests/renderer/components/commands-publish-button-spec.tsx

```diff
@@ -40,11 +40,11 @@ describe('Action button component', () => {
   const gistCreateOpts = {
     description,
     files: {
-      [DefaultEditorId.html]: { content: 'html-content' },
-      [DefaultEditorId.renderer]: { content: 'renderer-content' },
-      [DefaultEditorId.main]: { content: 'main-content' },
-      [DefaultEditorId.preload]: { content: 'preload-content' },
-      [DefaultEditorId.css]: { content: 'css-content' },
+      [DefaultEditorId.html]: { content: '<!-- index.html -->' },
+      [DefaultEditorId.renderer]: { content: '// renderer.js' },
+      [DefaultEditorId.main]: { content: '// main.js' },
+      [DefaultEditorId.preload]: { content: '// preload.js' },
+      [DefaultEditorId.css]: { content: '/* style.css */' },
```

Comment:
(making these values consistent with the ones made in the [`createEditorValues()`](https://github.com/electron/fiddle/blob/8a38d2a7a1811cf972494a267a714ca51d5e9553/tests/mocks/editor-values.ts#L3) test helper. Previous values came from the [`AppMock.getEditorValues jest.Mock`](https://github.com/electron/fiddle/blob/8a38d2a7a1811cf972494a267a714ca51d5e9553/tests/mocks/app.ts#L13)